### PR TITLE
improve prettier setup

### PR DESCRIPTION
**Pull request by hilary**
Fixes some issues we had with our prettier setup:
- sometimes prettier would try to reinitialize and add `lint-staged` (which adds `mrm`)
- the pre-commit githook wasn't running
- `pretty-quick` has `master` hard-coded as the default branch for github

### CODE REVIEW

**Review by senechko — APPROVED** [review]

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1 @@
+printWidth: 80
```
Comment (senechko) [inline .prettierrc:1]: 80 is the default so i dont think this is necessary 
image

https://prettier.io/docs/en/options.html

Comment (hilary) [reply .prettierrc:1]: I found that prettier seemed to ignore the presence of an empty file. :disappointed:

**Review by rogue0137 — COMMENTED** [review]
@hilary was the main reason for removing `pretty-quick` to get rid of it's reliance on using `master` as the default branch?

**Comment by hilary** [comment]
> @hilary was the main reason for removing `pretty-quick` to get rid of it's reliance on using `master` as the default branch?

The recommended design adding the `husky`/`pretty-quick` `JSON` stanza in `package.json` wasn't working. After some debugging, I noticed that `master` was built-in to `pretty-quick`. I did experiment `pretty-quick`s `--branch` option, but that didn't work either. At that point I looked at husky's docs again, saw that husky didn't have examples with stanzas in `package.json`, but used the traditional githook setup, breathed a sigh of relief and switched.

Looking today, it appears as if husky v4 supported stanzas in `package.json`, while husky v5 does not / no longer needs them / whatever. My normal inclination would be to research this until I understood every nuance, but I'm trying to stay on task. :stuck_out_tongue_closed_eyes:

**Review by rogue0137 — COMMENTED** [review]
testing

**Pull request merged**